templates/ez_segmenter.ipynb

```diff
@@ -0,0 +1,1019 @@
+{
```

Comment:
<p>Recommend changing to something like:</p><p> </p><pre class="ql-syntax" spellcheck="false">if not os.path.exists(base_dir):
  raise ValueError(f"The base_dir {base_dir} does not exist, please double check your path")
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/958/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Done</p>